docs/nerf_dataset_tips.md

```diff
@@ -129,6 +131,35 @@ With an >=iPhone 12 Pro, one can use [Record3D](https://record3d.app/) to collec
 	```
 	instant-ngp$ ./instant-ngp path/to/data
 	```
+	
+### NeRFCapture
+[NeRFCapture](https://github.com/jc211/NeRFCapture) is an iOS app that runs on any ARKit device. It allows you to stream images directly from your phone to InstantNGP thus enabling a more interactive experience. It can also collect an offline dataset for later use. 
+
+The following dependencies are needed to run the NeRFCapture script:
+```
+pip install cyclonedds
+pip install opencv-python
+pip install Pillow
```

Comment:
__instant-ngp__ already contains `imageio` in its requirements. Could you use it instead of `Pillow` for image saving to avoid the extra dependency? (From how I understand `nerfcapture2nerf.py`, that's the only use of `PIL`.)

Reply:
I don't know why but `imageio` is super slow when saving pngs. I used `opencv` instead.

Reply:
That's fair -- and good to know. TIL.